### ISSUE TITLE
Create Dataloaders for datasets on branch dataloaders

### ISSUE THREAD

**Issue by alexwdong** [alexwdong/IncubatorCVProject#9]
We need to create pytorch dataloaders for the datasets we have.

For each dataset, we need to create a (dataloader? or dataset?) class in pytorch that will load the data.

The dataloader should do some minor preprocessing on the images, specifically doing at least the following two transformations:

1) Padding the images with zeros, so that each image is square, but the aspect ratio of the content is preserved. (I'm also open to other ways to do this, since this seems like a potentially complicated part of the data processing).

2) Downsizing the padded images so that they all end up having the same size.

As an example, in the Stanford Dogs dataset, there are varying sizes of images (e.g, Image A has dimension (200x300), Image B has dimension (800 x 600). As far as I can tell (i just briefly looked), 'portrait' and 'landscape' orientations seem to be relatively evenly distributed, so both images would be padded with 0s so that they are square. 
Image A would be padded with 100 rows of 0s (50 on the top and 50 on the bottom), for a dimension of (300x300)
Image B would be padded with 200 columns of 0s (100 on the left and 100 on the right), for a final dimension of (800 x 800)

Both images would then be downsized to a smaller square image (Let's set 128x128 by default).

Our datasets that we have:

Stanford Dogs dataset: https://www.kaggle.com/jessicali9530/stanford-dogs-dataset

MIT Indoor Scenes: http://web.mit.edu/torralba/www/indoor.html

**Comment by alexwdong** [comment]
I've commited some code in the "dataloader" branch, under src/data_loader.py

The file has the transformations I requested, written by me. However, I learned that pytorch has its own image transformations? If pytorch has these image transformations written, then it would be preferable to use their functions.
